bin/_header.html

```diff
@@ -6,19 +6,46 @@
         <li class="name">
           <h1><a href="/">Home</a></h1>
         </li>
-        <li class="toggle-topbar menu-icon"><a href="#"><span>Menu</span></a></li>
       </ul>
-      <div class="top-bar-section">
-        <ul class="right">
-          <li class="has-dropdown"><a href="assignments.html">Assignments</a>
-            <ul class="dropdown">
-              <li><a href="#">First link in dropdown</a></li>
-            </ul>
-          </li>
-          <li><a href="news.html">News</a></li>
-          <li class="active"><a href="submit-tip.html">Submit Tip</a></li>
-        </ul>
-      </div>
+      <ul class="left">
+        <li class="name">
+          <h1><a href="/about.html">About</a></h1>
+        </li>
+      </ul>
     </nav>
   </header>
-</div>
\ No newline at end of file
+</div>
+<!--        li-->
```

Comment:
No need to comment these out. If we don't need them, I'd just say delete them. This is why we have version control in the first play ;)